editor/src/components/canvas/canvas-strategies/canvas-strategies.tsx

```diff
@@ -3,14 +3,19 @@ import { createSelector } from 'reselect'
 import { addAllUniquelyBy, mapDropNulls, sortBy } from '../../../core/shared/array-utils'
 import { ElementInstanceMetadataMap } from '../../../core/shared/element-template'
 import { arrayEquals } from '../../../core/shared/utils'
-import { InteractionSession, StrategyState } from './interaction-state'
 import { InnerDispatchResult } from '../../editor/store/dispatch'
 import { EditorStorePatched } from '../../editor/store/editor-state'
 import { useEditorState } from '../../editor/store/store-hook'
 import { CanvasCommand } from '../commands/commands'
+import { absoluteMoveStrategy } from './absolute-move-strategy'
+import { absoluteReparentStrategy } from './absolute-reparent-strategy'
 import { CanvasStrategy, ControlWithKey, InteractionCanvasState } from './canvas-strategy-types'
+import { InteractionSession, StrategyState } from './interaction-state'
```

Comment:
I took the liberty to run Organize Imports on some of these files, sorry for the noise, but I thought it's the right opportunity to sneak this in!